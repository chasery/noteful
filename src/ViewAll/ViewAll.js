import React, { Component, Fragment } from "react";
import STORE from "../store";
import FolderList from "../FolderList/FolderList";
import Folder from "../Folder/Folder";
import NoteList from "../NoteList/NoteList";
import Note from "../Note/Note";
import "./ViewAll.css";

class ViewAll extends Component {
  render() {
    return (
      <Fragment>
        <FolderList>
          <Folder id="test" name="Test Folder" />
          <Folder id="test" name="Test Folder" />
          <Folder id="test" name="Test Folder" />
        </FolderList>
        <NoteList>
          <Note id="test" name="Test Note" modified="10/10/2020" />
          <Note id="test" name="Test Note" modified="10/10/2020" />
          <Note id="test" name="Test Note" modified="10/10/2020" />
        </NoteList>
      </Fragment>
    );
  }
}

export default ViewAll;
