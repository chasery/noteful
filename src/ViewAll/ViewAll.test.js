import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import ViewAll from "./ViewAll";

describe("ViewAll component", () => {
  it("renders without error", () => {
    const wrapper = shallow(<ViewAll />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
