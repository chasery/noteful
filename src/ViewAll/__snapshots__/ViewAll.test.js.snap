// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewAll component renders without error 1`] = `
<Fragment>
  <FolderList>
    <Folder
      id="test"
      name="Test Folder"
    />
    <Folder
      id="test"
      name="Test Folder"
    />
    <Folder
      id="test"
      name="Test Folder"
    />
  </FolderList>
  <NoteList>
    <Note
      id="test"
      modified="10/10/2020"
      name="Test Note"
    />
    <Note
      id="test"
      modified="10/10/2020"
      name="Test Note"
    />
    <Note
      id="test"
      modified="10/10/2020"
      name="Test Note"
    />
  </NoteList>
</Fragment>
`;
